_pages/fsa/install/building-the-fsa-storefront-from-libraries-2-0.md

```diff
@@ -81,7 +81,7 @@ The easiest way to start a new project is to use Angular Schematics to set up yo
 You can add FSA Spartacus libraries to your Spartacus Angular project by running the following command from your project root:
 
 ```shell
-ng add @spartacus/fsa-schematics --baseSite=sample-financial-site --currency=usd,eur --language=en,de,fr
+ng add @spartacus/fsa-schematics --base-site=sample-financial-site --currency=usd,eur --language=en,de,fr
```

Comment:
Revert changes in this file. It's doc for an older version of Spartacus. 

Although it is not in one of the archived doc site version, the page says it's for spartacus 2